vignettes/lintr.Rmd

```diff
@@ -216,7 +216,18 @@ If you want to use all available linters *except* a few, you can exclude them us
 linters: linters_with_defaults(
     commented_code_linter = NULL,
     implicit_integer_linter = NULL,
-    defaults = linters_with_tags(tags = NULL)
+    defaults = all_linters()
+  )
+```
+
+Alternatively, you can also do:
+
+```r
+linters: linters_with_defaults(
```

Comment:
What I had in mind was

```r
linters: all_linters(
    commented_code_linter = NULL,
    implicit_integer_linter = NULL
  )
```

Reply:
Ah, I see. 

I've changed the example.